dotCMS/src/main/java/com/dotcms/browser/BrowserAPIImpl.java

```diff
@@ -54,7 +58,7 @@ public class BrowserAPIImpl implements BrowserAPI {
     private final FolderAPI folderAPI = APILocator.getFolderAPI();
     private final PermissionAPI permissionAPI = APILocator.getPermissionAPI();
     private final ShortyIdAPI shortyIdAPI = APILocator.getShortyAPI();
-
+    private final ESContentletAPIImpl esContentletAPI = new ESContentletAPIImpl();
```

Comment:
Use the `APILocator.getContentletAPI()` instead.

Reply:
`APILocator.getContentletAPI()` will not work here

![image](https://user-images.githubusercontent.com/111717530/225634079-9635d689-9da0-4711-8103-1b5f21eab68c.png)

Reply:
by the way I have added this method in the APILocator.java
![image](https://user-images.githubusercontent.com/111717530/225637824-c5c74919-2389-430e-92bd-3e413ffced4c.png)